### ISSUE TITLE
🔧 [CHANGE] - Add Autostart and Autostop Machine Options to `fly.toml` or Documentation

### ISSUE THREAD

**Issue by Avi-Lance** [actualbudget/docs#416]
### Description

Fly.io has recently updated their pricing. You can find more details in this issue: [#415](https://github.com/actualbudget/docs/issues/415).

To help reduce costs, you can set the following two variables in the `fly.toml` file:
```
[[services]]
auto_start_machines = true
auto_stop_machines = true
 ```

There are a couple of ways to address this change:
1.     We could update the default fly.toml to include these options.
2.     Alternatively, we could document these options so that users can add them to their configuration themselves.

### Expected behavior

With these options, Fly.io automatically starts an instance if there are web requests and shuts it down if there haven’t been any requests for a few minutes.

Based on my limited testing, this may delay the first connection to Actual Budget by 5-15 seconds if there isn’t a cached copy in the browser. However, if there is a cached copy—such as when the Actual Budget Progressive Web App is installed on your phone—the difference in loading time is less noticeable.

**Comment by RubenOlsen** [comment]
If you are comfortable editing the documentation yourself, could you add this?

**Comment by Avi-Lance** [comment]
I added the changes in this PR: https://github.com/actualbudget/docs/pull/418

**Comment by matt-fidd** [comment]
Fixed in #418
